<launch>
  <!-- <arg
    name="pcl_sub_topic"
    default = "transformed_cloud"
  />
  <node pkg="pcl_filter" name="velodyne_tf_pub" type="velodyne_tf_pub"
        output="screen" />
  <node pkg="pcl_filter" name="dji_tf_pub" type="dji_tf_pub"
        output="screen" />
  <node pkg="pcl_filter" name="odom_pub" type="odom_pub"
        output="screen" /> -->

  <node pkg="pcl_filter" name="transform_cloud_to_cam_frame" type="transform_cloud_to_cam_frame" output="screen">
	<!--<remap from = "/camera/rgb/image_raw" to = "/image_mask"/>-->
  </node>

  <!-- <node pkg="pcl_filter" name="transform_cloud_velo_pub" type="transform_cloud_velo_pub" -->
        <!-- output="screen" /> -->
  <!-- <node pkg="pcl_filter" name="transform_cloud_pub" type="transform_cloud_pub" -->
        <!-- output="screen" /> -->
  <!-- <node pkg="pcl_filter" name="pcl_filter" type="pcl_filter"
        output="screen" args="$(arg pcl_sub_topic)"/> -->
  <!-- <node pkg="pcl_filter" name="register_multiple" type="register_multiple" -->
        <!-- output="screen" /> -->
  <!-- <node pkg="pcl_filter" name="register_multiple_velo" type="register_multiple_velo" -->
        <!-- output="screen" /> -->

  <!-- trimmed octomap server -->
  <node pkg="octomap_server" type="octomap_server_node_trimmed" name="octomap_server_trimmed">
  <!-- resolution 2 -->
    <param name="resolution" value="2" />
	<!-- fixed map frame (set to 'map' if SLAM or localization running!) -->
    <param name="frame_id" type="string" value="world" />
	<!-- maximum range to integrate (speedup!) -->
    <param name="sensor_model/max_range" value="40" />
	<!-- data source to integrate (PointCloud2) -->
    <remap from="cloud_in" to="transformed_cloud_image_frame" />
  </node>

  <!-- velodyne point cloud transformation to octomap -->
  <!-- <node pkg="octomap_server" type="octomap_server_node" name="octomap_server_velo"> -->
    <!-- <param name="resolution" value="1" /> -->
	<!-- fixed map frame (set to 'map' if SLAM or localization running!) -->
    <!-- <param name="frame_id" type="string" value="world" /> -->
	<!-- maximum range to integrate (speedup!) -->
    <!-- <param name="sensor_model/max_range" value="40" /> -->
	<!-- data source to integrate (PointCloud2) -->
    <!-- <remap from="cloud_in" to="velodyne_points" /> -->
  <!-- </node> -->

</launch>
